src/Components/CartItems.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+
+function CartItems(){
+    return(  <div className="ItemsInCart"
+    style={{ display: "flex", flexDirection: "column", gap : "1rem" }}
+  >
+    <div className="horizontal-card">
+      <img src="https://picsum.photos/308/390" alt="" srcset="" />
+      <div className="card-content">
+        <span className="product-name">product name</span>
+        <h2 className="product-card-amount">XXXX$</h2>
+
+        <div className="quantity-div">
+          <span>Quantity :</span>
+          <button className="increase-qty-btn">-</button>
+          <input type="number" name="" id="" value="0" />
```

Comment:
you may avoid giving empty values to the attributes

Reply:
I missed them.. thanks...I'll delete it